src/AddressRegistry.Api.Extract/Extracts/AddressRegistryExtractBuilder.cs

```diff
@@ -144,8 +144,6 @@ byte[] TransformRecord(AddressExtractItemV2 r)
 
                 switch (municipality.PrimaryLanguage)
                 {
-                    case null:
-                    case Taal.NL:
```

Comment:
This is of course correct, but I like expressing: I know `NL` exists and it should be like default behavior. So if someone comes and looks at the code it answers: have you forgot `NL`?